apisix/plugins/ip-restriction.lua

```diff
@@ -36,6 +36,10 @@ local schema = {
                     items = {anyOf = core.schema.ip_def},
                     minItems = 1
                 },
+                message = {
```

Comment:
Better to extract this field to avoid repeating it

Reply:
And add some limitations for the max length?

Reply:
I'll figure out how to modify it, thanks

Reply:
See https://github.com/apache/apisix/blob/07bfe5b9c03ad361cb71541eb02fd661f1323213/apisix/schema_def.lua#L33

Reply:
ok👌